### PR TITLE
added console.log

### PR DESCRIPTION
**Pull request by GuyDahn**
Give me the clone!

### CODE REVIEW

**Comment by Rachel-Aizenman** [comment]
here it is!

**Comment by Rachel-Aizenman** [comment]
okkkk

File: logic.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const CoonGame = function (width, height) {
   const removeCoon = function (coonID) {
     const index = _coons.findIndex(i => i.id === coonID)
     _coons.splice(index, 1)
+    console.log("good job!")
```
Comment (GuyDahn) [inline logic.js]: EXQUISITE PRINT CODE, OUTSTANDING!

Comment (Rachel-Aizenman) [reply logic.js]: thanks

**Pull request merged**